hail/python/hail/experimental/datasets.py

```diff
@@ -0,0 +1,294 @@
+
+import json
+import hail as hl
+
+
+def load_dataset(dataset_name,
+                 reference_genome,
+                 config_file='gs://hail-datasets/hail_datasets.config.json'):
+    """Load a Hail-formatted genetic dataset.
+
+    Example
+    -------
+
+    >>> # Load 1000 Genomes chromosome X MatrixTable with GRCh38 coordinates
+    >>> mt_1kg = hl.experimental.load_dataset('1000_genomes_phase3_chrX',   # doctest: +SKIP
+    ...                                       reference_genome='GRCh38')
+
+    >>> # Count rows and columns
+    >>> mt_1kg.count()  # doctest: +SKIP
+
+    .. code-block:: text
+
+        (3474116, 2504)
+
+    >>> # Inspect contents of the MatrixTable
+    >>> mt_1kg.describe()   # doctest: +SKIP
```

Comment:
I think we should delete this example, it's going to make the doc pretty unreadable given the length of the schema. Instead, let's just link to the datasets page to see a list of available things (we should definitely have a link to that page from this function)